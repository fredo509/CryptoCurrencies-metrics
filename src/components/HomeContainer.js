import React from 'react';
// import HomeList from './HomeList';

function HomeContainer() {
  return (
    <div className="home">
      {/* <HomeList /> */}
      <p>List comming soon</p>
    </div>
  );
}

export default HomeContainer;
