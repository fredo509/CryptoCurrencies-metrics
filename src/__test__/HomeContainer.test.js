import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import HomeContainer from '../components/Home/HomeContainer';
import store from '../Redux/store';

it('Test HomeContainer renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <HomeContainer />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
