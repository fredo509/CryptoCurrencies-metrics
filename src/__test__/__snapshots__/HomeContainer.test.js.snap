// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HomeContainer renders correctly 1`] = `
<div
  className="home"
>
  <ul
    className="homeList"
  />
</div>
`;
